blog/2025-09-25-conda-pypi.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: "Conda ≠ PyPI: Why Conda Is More Than a Package Manager"
+slug: "conda-pypi"
+authors: [dbast, jezdez]
+tags: [conda, packaging, distribution]
+description: "Part 1 of the 'Conda Is Not PyPI' series—why conda is a multi-language user-space distribution, not just a Python package manager."
+---
+
+*Part 1 of our series “Conda Is Not PyPI: Understanding Conda as a User-Space Distro”.*
+
+<!-- truncate -->
+
+---
+
+When people first encounter conda, the most common misconception is:
+
+> *“Isn’t conda just another Python package manager, like pip?”*
+
+The short answer: **no**.
+
+Pip (and npm, RubyGems, Cargo, etc.) are **language-specific library registries**. Conda, on the other hand, is a **multi-language, user-space distribution** that can assemble coherent runtime and build environments across programming languages, system libraries, compilers, and tools.
```

Comment:
```suggestion
PyPI (and npm, RubyGems, Cargo, etc.) are **language-specific library registries**. Conda, on the other hand, is a **multi-language, user-space distribution** that can assemble coherent runtime and build environments across programming languages, system libraries, compilers, and tools.
```

Reply:
We need to be careful to not conflate pip with PyPI.

Reply:
https://github.com/conda/conda-dot-org/pull/250/commits/97b84ac9d425be760fb14849b5db7bfdd0035abe